modules/lib/src/main/resources/assets/js/app/inputtype/text/HtmlArea.ts

```diff
@@ -622,8 +623,11 @@ export class HtmlArea
         super.updateInputOccurrenceElement(textAreaEl, property, unchangedOnly);
     }
 
-    isAiEditable(): boolean {
-        return true;
+    getAiConfig(): AiConfig {
+        return {
+            group: this.getContext().formContext?.getName(),
+            aiTools: this.getContext().formContext?.getAiTools() ?? [],
```

Comment:
Don't forget moving tools from Array to Set, if you'll make corresponding changes to libAdminUI.